wandb/sdk/artifacts/artifact.py

```diff
@@ -510,11 +498,13 @@ def metadata(self, metadata: dict) -> None:
 
     @property
     def ttl(self) -> Union[timedelta, None]:
-        """Time To Live (TTL).
+        """The time-to-live (TTL) policy of an artifact.
```

Comment:
```suggestion
        """The time-to-live (TTL) policy of an artifact.
```
I'm a bit confused if this is a method that sets the ttl or returns the property. @ngrayluna  can you confirm? It looks like the property since it doesn't accept a TTL duration but I cannot find a method in here for setting a TTL.

Reply:
This `ttl()` method only returns the value that exists on the artifact (It has the `@property` decorator above it). You can add the docstrings to add ttl under the `ttl()` method that has the `@setter` decorator above it

Reply:
gotcha - thank you, I will look for this method. Noted this is just a property.